py/tests/integration/common.py

```diff
@@ -46,57 +47,36 @@ def node_online(api):
     except Exception as e:
         return False
 
-def start_node(name):
-    os.system("make " + name + "-start")
+def start_node(name, config_filename=None):
+    config_prefix = ""
+    if config_filename != None:
+        if config_filename[0] == "/": # absolute path
+            config_prefix =  'ERL_FLAGS="-config ' + config_filename + '" ' 
+        else:
+            config_prefix =  'ERL_FLAGS="-config `pwd`/' + config_filename + '" ' 
+    os.system(config_prefix + "make " + name + "-start")
     api = external_api(name)
-    wait(lambda: node_online(api), timeout_seconds=3, sleep_seconds=0.5)
+    wait(lambda: node_online(api), timeout_seconds=10, sleep_seconds=0.5)
 
 def stop_node(name):
     os.system("make " + name + "-stop")
+    time.sleep(3)
```

Comment:
Use a wait here instead

Reply:
well yes, but `wait` expects a predicate - what should we be waiting for? I don't think `ps -fea| grep epoch | wc -l ` is good way of validation that the node is dead

Reply:
I guess you could check that it doesn't respond to ping, but that might succeed too early.

Nevermind. Leave it as is for now.